Cargo.toml

```diff
@@ -23,3 +23,155 @@ members = [
     "sugondat/shim/common/sovereign",
     "sugondat/subxt-autogen"
 ]
+
+[workspace.package]
+authors = ["Robert Habermeier - @rphmeier" , "Sergei Shulepov - @pepyakin", "Gabriele Miotti - @gabriele-0201"]
```

Comment:
A couple of things. I think this field is conventionally has the same format as name and email.

Quote from Cargo docs:

```
[package]
# ...
authors = ["Graydon Hoare", "Fnu Lnu <no-reply@rust-lang.org>"]
```

However, I would rather not put any names here. It's just not scalable, may introduce some unnecessary feelings, and most importantly serves no purpose! 

I see the following ways:

1. remove it altogether. I don't think it's required.
2. use unpersonified email, similar to what we do in Parity. smth like `parity developers <admin@parity.io>`.